TypeCobol.LanguageServer/Workspace.cs

```diff
@@ -153,14 +157,16 @@ public Workspace(string rootDirectoryFullName, string workspaceName, Queue<Messa
             _openedDocuments = new Dictionary<Uri, DocumentContext>();
             _fileCompilerWaittingForNodePhase = new List<FileCompiler>();
             _Logger = logger;
+            this.AnalyzerProvider = new AnalyzerProvider();
+            this.AnalyzerProvider.AddActivator((o, t) => CfgDfaAnalyzerFactory.CreateCfgAnalyzer(TypeCobolLanguageServer.lspcfgId, CfgBuildingMode.Standard));
 
             this._rootDirectoryFullName = rootDirectoryFullName;
             this._workspaceName = workspaceName;
 
             var defaultDocumentFormat = new DocumentFormat(Encoding.GetEncoding("iso-8859-1"), EndOfLineDelimiter.CrLfCharacters, 80, ColumnsLayout.CobolReferenceFormat);
             this.CompilationProject = new CompilationProject(
                 _workspaceName, _rootDirectoryFullName, Helpers.DEFAULT_EXTENSIONS, defaultDocumentFormat,
-                new TypeCobolOptions(), null); //Initialize a default CompilationProject - has to be recreated after ConfigurationChange Notification
+                new TypeCobolOptions(), this.AnalyzerProvider); //Initialize a default CompilationProject - has to be recreated after ConfigurationChange Notification
```

Comment:
I thought the initial `CompilationProject` would not require any analyzerProvider because it will be recreated anyway when we get the client configuration (including the cfg/dfa options). So better keep it to null initially.

Reply:
Right, and you make me discover multi analyzer instantiation.